### ISSUE TITLE
wrong type of datapoints after upgrade

### ISSUE THREAD

**Issue by metaxa1** [iobroker-community-adapters/ioBroker.synology#7]
<html>
<body>
<p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px !important; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">I'm sorry, but after updating to v1.1.1, the same log entries sometimes appear again.<br style="box-sizing: border-box;">I stopped the adapter and deleted all data points on the objects. The v1.1.1. installed via GitHub Link over it and restarted the adapter 2x. What did 1 wrong?</p><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">`</p>

synology.0 | 2021-08-09 11:48:00.704 | info | (30961) State value to set for "synology.0.DiskStationManager.vol_info.volume_1.used_size" has to be type "number" but received type "string"
-- | -- | -- | --
synology.0 | 2021-08-09 11:48:00.654 | info | (30961) State value to set for "synology.0.DiskStationManager.vol_info.volume_1.total_size" has to be type "number" but received type "string"
synology.0 | 2021-08-09 11:48:00.653 | info | (30961) State value to set for "synology.0.DiskStationManager.hdd_info.sata_drive_4.capacity" has to be type "number" but received type "string"
synology.0 | 2021-08-09 11:48:00.648 | info | (30961) State value to set for "synology.0.DiskStationManager.hdd_info.sata_drive_4.ebox_order" has to be type "string" but received type "number"
synology.0 | 2021-08-09 11:48:00.646 | info | (30961) State value to set for "synology.0.DiskStationManager.hdd_info.sata_drive_3.capacity" has to be type "number" but received type "string"
synology.0 | 2021-08-09 11:48:00.645 | info | (30961) State value to set for "synology.0.DiskStationManager.hdd_info.sata_drive_3.ebox_order" has to be type "string" but received type "number"
synology.0 | 2021-08-09 11:48:00.640 | info | (30961) State value to set for "synology.0.DiskStationManager.hdd_info.sata_drive_2.capacity" has to be type "number" but received type "string"
synology.0 | 2021-08-09 11:48:00.639 | info | (30961) State value to set for "synology.0.DiskStationManager.hdd_info.sata_drive_2.ebox_order" has to be type "string" but received type "number"
synology.0 | 2021-08-09 11:48:00.631 | info | (30961) State value to set for "synology.0.DiskStationManager.hdd_info.sata_drive_1.capacity" has to be type "number" but received type "string"
synology.0 | 2021-08-09 11:48:00.630 | info | (30961) State value to set for "synology.0.DiskStationManager.hdd_info.sata_drive_1.ebox_order" has to be type "string" but received type "number"
synology.0 | 2021-08-09 11:48:00.624 | info | (30961) State value to set for "synology.0.DiskStationManager.info.upgrade_ready" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:48:00.623 | info | (30961) State value to set for "synology.0.DiskStationManager.info.is_system_crashed" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:48:00.621 | info | (30961) State value to set for "synology.0.DiskStationManager.info.memory_size" has to be type "string" but received type "number"
synology.0 | 2021-08-09 11:47:42.968 | info | (30961) State value to set for "synology.0.FileStation.info.uid" has to be type "string" but received type "number"
synology.0 | 2021-08-09 11:47:42.966 | info | (30961) State value to set for "synology.0.FileStation.info.support_virtual.enable_remote_mount" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:47:42.964 | info | (30961) State value to set for "synology.0.FileStation.info.support_virtual.enable_iso_mount" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:47:42.963 | info | (30961) State value to set for "synology.0.FileStation.info.support_vfs" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:47:42.962 | info | (30961) State value to set for "synology.0.FileStation.info.support_sharing" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:47:42.961 | info | (30961) State value to set for "synology.0.FileStation.info.support_file_request" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:47:42.959 | info | (30961) State value to set for "synology.0.FileStation.info.items.2" has to be type "string" but received type "object"
synology.0 | 2021-08-09 11:47:42.958 | info | (30961) State value to set for "synology.0.FileStation.info.items.1" has to be type "string" but received type "object"
synology.0 | 2021-08-09 11:47:42.925 | info | (30961) State value to set for "synology.0.FileStation.info.items.0" has to be type "string" but received type "object"
synology.0 | 2021-08-09 11:47:42.918 | info | (30961) State value to set for "synology.0.FileStation.info.is_manager" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:47:42.916 | info | (30961) State value to set for "synology.0.FileStation.info.enable_list_usergrp" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:47:42.914 | info | (30961) State value to set for "synology.0.DiskStationManager.info.temperature_warn" has to be type "string" but received type "boolean"
synology.0 | 2021-08-09 11:47:41.584 | info | (30961) DSM 6
synology.0 | 2021-08-09 11:47:36.166 | info | (30961) Connecting to Synology 192.168.0.98:5000
synology.0 | 2021-08-09 11:47:36.060 | info | (30961) starting. Version 1.1.1 in /opt/iobroker/node_modules/iobroker.synology, node: v12.22.2, js-controller: 3.3.15
synology.0 | 2021-08-09 11:47:12.205 | info | (24661) Terminated (ADAPTER_REQUESTED_TERMINATION): Without reason

</body>
</html>

**Comment by DcFamas** [comment]
Same here!

**Comment by MeisterTR** [comment]
fixed in next version 1.1.2

**Comment by MMeinhardt1** [comment]
Hi,
with V1.1.3 i have the following log entries.



```
2021-08-24 19:56:53.230 - info: synology.0 (3011) starting. Version 1.1.3 in /opt/iobroker/node_modules/iobroker.synology, node: v12.22.5, js-controller: 3.3.15
2021-08-24 19:56:53.281 - info: synology.0 (3011) Connecting to Synology 192.168.178.20:5001
2021-08-24 19:56:57.436 - info: synology.0 (3011) DSM 1
2021-08-24 19:56:58.960 - info: synology.0 (3011) State value to set for "synology.0.DiskStationManager.info.temperature_warn" has to be type "string" but received type "boolean"
2021-08-24 19:56:58.962 - info: synology.0 (3011) State value to set for "synology.0.DiskStationManager.info.uptime" has to be type "string" but received type "number"
2021-08-24 19:56:58.964 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.enable_list_usergrp" has to be type "string" but received type "boolean"
2021-08-24 19:56:58.966 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.is_manager" has to be type "string" but received type "boolean"
2021-08-24 19:56:58.968 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.items.0" has to be type "string" but received type "object"
2021-08-24 19:56:58.969 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.items.1" has to be type "string" but received type "object"
2021-08-24 19:56:58.971 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.items.2" has to be type "string" but received type "object"
2021-08-24 19:56:58.973 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.support_file_request" has to be type "string" but received type "boolean"
2021-08-24 19:56:58.975 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.support_sharing" has to be type "string" but received type "boolean"
2021-08-24 19:56:58.993 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.support_vfs" has to be type "string" but received type "boolean"
2021-08-24 19:56:58.994 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.support_virtual.enable_iso_mount" has to be type "string" but received type "boolean"
2021-08-24 19:56:58.996 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.support_virtual.enable_remote_mount" has to be type "string" but received type "boolean"
2021-08-24 19:56:59.006 - info: synology.0 (3011) State value to set for "synology.0.FileStation.info.uid" has to be type "string" but received type "number"
```

Many thanks

**Comment by MMeinhardt1** [comment]
Update:
After stopping the Adapter, delete object tree and restart adapter the following entries are in the log.



```
2021-08-24 20:08:00.181 - info: synology.0 (21416) starting. Version 1.1.3 in /opt/iobroker/node_modules/iobroker.synology, node: v12.22.5, js-controller: 3.3.15
2021-08-24 20:08:00.222 - info: synology.0 (21416) Connecting to Synology 192.168.178.20:5001
2021-08-24 20:08:03.709 - info: synology.0 (21416) DSM 1
2021-08-24 20:08:05.281 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.codepage" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.285 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.model" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.288 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.ram" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.292 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.serial" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.296 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.temperature" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.298 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.temperature_warn" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.300 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.time" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.302 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.uptime" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.305 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.version" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.307 - warn: synology.0 (21416) State "synology.0.DiskStationManager.info.version_string" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.309 - warn: synology.0 (21416) State "synology.0.FileStation.info.enable_list_usergrp" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.311 - warn: synology.0 (21416) State "synology.0.FileStation.info.hostname" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.314 - warn: synology.0 (21416) State "synology.0.FileStation.info.is_manager" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.315 - warn: synology.0 (21416) State "synology.0.FileStation.info.items.0" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.317 - warn: synology.0 (21416) State "synology.0.FileStation.info.items.1" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.320 - warn: synology.0 (21416) State "synology.0.FileStation.info.items.2" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.322 - warn: synology.0 (21416) State "synology.0.FileStation.info.support_file_request" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.324 - warn: synology.0 (21416) State "synology.0.FileStation.info.support_sharing" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.326 - warn: synology.0 (21416) State "synology.0.FileStation.info.support_vfs" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.329 - warn: synology.0 (21416) State "synology.0.FileStation.info.support_virtual.enable_iso_mount" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.331 - warn: synology.0 (21416) State "synology.0.FileStation.info.support_virtual.enable_remote_mount" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.333 - warn: synology.0 (21416) State "synology.0.FileStation.info.support_virtual_protocol.0" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.336 - warn: synology.0 (21416) State "synology.0.FileStation.info.support_virtual_protocol.1" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.338 - warn: synology.0 (21416) State "synology.0.FileStation.info.support_virtual_protocol.2" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.339 - warn: synology.0 (21416) State "synology.0.FileStation.info.system_codepage" has no existing object, this might lead to an error in future versions
2021-08-24 20:08:05.366 - warn: synology.0 (21416) State "synology.0.FileStation.info.uid" has no existing object, this might lead to an error in future versions
```

**Comment by Apollon77** [comment]
@MeisterTR ?

@MMeinhardt1 For the future: adding posts to a closed issue normally makes NO sense. The way is to open an enw issue

**Comment by weindler** [comment]
Hi, i have the same, after update to 1.1.3 there are 99 warn infos of this error
`
<html><body>


synology.0 | 2021-09-16 17:00:39.747 | warn | State "synology.0.FileStation.sharing.list" has no existing object, this might lead to an error in future versions
-- | -- | -- | --
synology.0 | 2021-09-16 17:00:39.746 | warn | State  "synology.0.DiskStationManager.vol_info.volume_1.desc" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.744 | warn | State  "synology.0.DiskStationManager.vol_info.volume_1.used" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.742 | warn | State  "synology.0.DiskStationManager.vol_info.volume_1.used_size" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.740 | warn | State  "synology.0.DiskStationManager.vol_info.volume_1.total_size" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.721 | warn | State  "synology.0.DiskStationManager.vol_info.volume_1.status" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.720 | warn | State  "synology.0.DiskStationManager.vol_info.volume_1.name" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.718 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_2.capacity" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.700 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_2.storage_pool" has  no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.699 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_2.temperature" has  no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.698 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_2.ebox_order" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.696 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_2.overview_status"  has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.693 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_2.model" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.692 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_2.diskno" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.683 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_1.capacity" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.653 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_1.storage_pool" has  no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.653 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_1.temperature" has  no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.652 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_1.ebox_order" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.642 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_1.overview_status"  has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.607 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_1.model" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.606 | warn | State  "synology.0.DiskStationManager.hdd_info.sata_drive_1.diskno" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.605 | warn | State "synology.0.DiskStationManager.info.upgrade_ready" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.604 | warn | State  "synology.0.DiskStationManager.info.is_system_crashed" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.603 | warn | State "synology.0.DiskStationManager.info.memory_size" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.602 | warn | State "synology.0.DiskStationManager.info.memory_usage" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:39.601 | warn | State "synology.0.DiskStationManager.info.cpu_load" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.608 | warn | State "synology.0.VideoStation.info.version_string" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.514 | warn | State "synology.0.VideoStation.info.version" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.513 | warn | State  "synology.0.VideoStation.info.transcoding_capability.transcode" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.512 | warn | State   "synology.0.VideoStation.info.transcoding_capability.software_transcode"  has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.456 | warn | State  "synology.0.VideoStation.info.transcoding_capability.remux" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.455 | warn | State   "synology.0.VideoStation.info.transcoding_capability.hardware_transcode"  has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.454 | warn | State   "synology.0.VideoStation.info.transcoding_capability.fhd_hardware_transcode"  has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.453 | warn | State  "synology.0.VideoStation.info.transcoding_capability.dtv_transcode" has  no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.452 | warn | State "synology.0.VideoStation.info.timezone_offset" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.451 | warn | State "synology.0.VideoStation.info.timezone" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.451 | warn | State "synology.0.VideoStation.info.privilege.sharing" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.450 | warn | State "synology.0.VideoStation.info.privilege.renderer" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.449 | warn | State  "synology.0.VideoStation.info.privilege.offline_conversion" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.448 | warn | State "synology.0.VideoStation.info.privilege.dtv" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.447 | warn | State  "synology.0.VideoStation.info.is_subtitle_search_enabled" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.446 | warn | State "synology.0.VideoStation.info.is_manager" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.443 | warn | State "synology.0.VideoStation.info.is_dtv_enabled" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.442 | warn | State "synology.0.AudioStation.info.version_string" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.441 | warn | State "synology.0.AudioStation.info.version" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.440 | warn | State  "synology.0.AudioStation.info.transcode_capability.1" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.439 | warn | State  "synology.0.AudioStation.info.transcode_capability.0" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.438 | warn | State  "synology.0.AudioStation.info.support_virtual_library" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.437 | warn | State "synology.0.AudioStation.info.support_usb" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.418 | warn | State "synology.0.AudioStation.info.support_bluetooth" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.417 | warn | State "synology.0.AudioStation.info.sid" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.416 | warn | State  "synology.0.AudioStation.info.settings.transcode_to_mp3" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.415 | warn | State  "synology.0.AudioStation.info.settings.prefer_using_html5" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.414 | warn | State  "synology.0.AudioStation.info.settings.enable_download" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.413 | warn | State  "synology.0.AudioStation.info.settings.disable_upnp" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.412 | warn | State  "synology.0.AudioStation.info.settings.audio_show_virtual_library" has  no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.411 | warn | State "synology.0.AudioStation.info.serial_number" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.409 | warn | State "synology.0.AudioStation.info.same_subnet" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.409 | warn | State "synology.0.AudioStation.info.remote_controller" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.408 | warn | State  "synology.0.AudioStation.info.privilege.upnp_browse" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.407 | warn | State "synology.0.AudioStation.info.privilege.tag_edit" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.406 | warn | State "synology.0.AudioStation.info.privilege.sharing" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.405 | warn | State  "synology.0.AudioStation.info.privilege.remote_player" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.390 | warn | State  "synology.0.AudioStation.info.privilege.playlist_edit" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.371 | warn | State "synology.0.AudioStation.info.playing_queue_max" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.370 | warn | State "synology.0.AudioStation.info.is_manager" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.368 | warn | State "synology.0.AudioStation.info.has_music_share" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.367 | warn | State "synology.0.AudioStation.info.enable_user_home" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.366 | warn | State  "synology.0.AudioStation.info.enable_personal_library" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.365 | warn | State "synology.0.AudioStation.info.enable_equalizer" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.363 | warn | State  "synology.0.AudioStation.info.dsd_decode_capability" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.362 | warn | State  "synology.0.AudioStation.info.browse_personal_library" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.361 | warn | State "synology.0.DownloadStation.info.version_string" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.360 | warn | State "synology.0.DownloadStation.info.version" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.359 | warn | State "synology.0.DownloadStation.info.is_manager" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.358 | warn | State "synology.0.FileStation.info.uid" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.357 | warn | State "synology.0.FileStation.info.system_codepage" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.356 | warn | State  "synology.0.FileStation.info.support_virtual_protocol.2" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.350 | warn | State  "synology.0.FileStation.info.support_virtual_protocol.1" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.348 | warn | State  "synology.0.FileStation.info.support_virtual_protocol.0" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.347 | warn | State  "synology.0.FileStation.info.support_virtual.enable_remote_mount" has  no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.346 | warn | State  "synology.0.FileStation.info.support_virtual.enable_iso_mount" has no  existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.345 | warn | State "synology.0.FileStation.info.support_vfs" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.344 | warn | State "synology.0.FileStation.info.support_sharing" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.343 | warn | State "synology.0.FileStation.info.support_file_request" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.341 | warn | State "synology.0.FileStation.info.items.1" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.340 | warn | State "synology.0.FileStation.info.items.0" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.339 | warn | State "synology.0.FileStation.info.is_manager" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.338 | warn | State "synology.0.FileStation.info.hostname" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.337 | warn | State "synology.0.FileStation.info.enable_list_usergrp" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.335 | warn | State "synology.0.DiskStationManager.info.version_string" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.334 | warn | State "synology.0.DiskStationManager.info.version" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.333 | warn | State "synology.0.DiskStationManager.info.uptime" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.331 | warn | State "synology.0.DiskStationManager.info.time" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.330 | warn | State  "synology.0.DiskStationManager.info.temperature_warn" has no existing  object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.329 | warn | State "synology.0.DiskStationManager.info.temperature" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.328 | warn | State "synology.0.DiskStationManager.info.serial" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.326 | warn | State "synology.0.DiskStationManager.info.ram" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.325 | warn | State "synology.0.DiskStationManager.info.model" has no existing object, this might lead to an error in future versions
synology.0 | 2021-09-16 17:00:35.323 | warn | State "synology.0.DiskStationManager.info.codepage" has no existing object, this might lead to an error in future versions


</body>
</html>`

I have cleared the full synology objects tree, but after i start the instances of synology the same errors are coming.

**Comment by MeisterTR** [comment]
fixed in 2.0.1
